src/screens/Accounts.js

```diff
@@ -93,7 +94,9 @@ class AccountCard extends PureComponent<*, *> {
   };
   render() {
     const { account, onPress } = this.props;
-    const data = getBalanceHistory(account, 30);
+    const data = getBalanceHistory(account, 10);
+    const startPrice: number = data[0].value;
+    const endPrice: number = data[data.length - 1].value;
```

Comment:
good practice here. we will refactor more things according to this idea everything is already in the balance history data 🚀

is it possible to not type `: number` here ? if it does infer properly, my general practice is to not type the local variables